### DISCUSSION TITLE
Docker image for testing meilisearch performance

### DISCUSSION THREAD

**Discussion by foremtehan** [meilisearch/product#534] [Feedback & Feature Proposal]
It world be great if there was a docker image that imports millions of fake data so that i play with meilisearch features and its performance

**Comment by curquiza** [comment]
Hello @foremtehan, I moved this discussion to the product repo so that if more person wants this image, they can upvote it.

Saying this, since Meilisearch is opensource, anyone is free to provide this kind of image on their own, we would love to promote it 😄
